canvas_sdk/v1/data/questionnaire.py

```diff
@@ -143,7 +143,7 @@ class Meta:
     id = models.UUIDField()
     dbid = models.BigIntegerField(primary_key=True)
     deleted = models.BooleanField()
-    committer = models.ForeignKey(CanvasUser, on_delete=models.DO_NOTHING)
+    committer = models.ForeignKey(CanvasUser, on_delete=models.DO_NOTHING, null=True)
```

Comment:
We need to do a mass change where we fix all of the missing `null=True` instances.